### ISSUE TITLE
CVE-2021-31535

### ISSUE THREAD

**Issue by deanrecord** [zulu-openjdk/zulu-openjdk#97]
Affected: azul/zulu-openjdk-alpine:15.0.1 (at least)
Found with AWS ECR Scan.

https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-31535

**Comment by ClaudioConsolmagno** [comment]
Came here looking for this. CVE also affects azul/zulu-openjdk-alpine:11.0.11
Found with [trivy scan](https://github.com/aquasecurity/trivy):
```
+---------+------------------+----------+-------------------+---------------+
| LIBRARY | VULNERABILITY ID | SEVERITY | INSTALLED VERSION | FIXED VERSION |
+---------+------------------+----------+-------------------+---------------+
| libx11  | CVE-2021-31535   | CRITICAL | 1.7.0-r0          | 1.7.1-r0      |
+---------+------------------+----------+-------------------+---------------+
```

**Comment by brunanazario** [comment]
I have the same alert.
I'm using azul/zulu-openjdk-alpine:15-jre.

**Comment by ClaudioConsolmagno** [comment]
I believe this is fixed with 11.0.12 (can't say for other jdk versions) but now there's a different CVE affecting that: https://github.com/zulu-openjdk/zulu-openjdk/issues/106

**Comment by ironstas1k** [comment]
Hi everyone,

We'v implemented continuous process for rebuilding 3 latest Zulu releases weekly. As I see now this vulnerability is gone for  azul/zulu-openjdk-alpine:11.0.11.
